components/form/index.en-US.md

```diff
@@ -275,3 +275,55 @@ validator(rule, value, callback) => {
   }
 }
 ```
+
+### How you can use `wrappedComponentRef` to get form instance in functional component
+
+you have to combine `forwardRef` with `useImperativeHandle` so you can get form instance by using `wrappedComponentRef`:
+
+```tsx
+import React, { createRef, forwardRef, useImperativeHandle, useState } from 'react';
+import { Form, Input, Button } from 'antd';
+import { FormComponentProps } from 'antd/lib/form/Form';
+
+interface FCFormProps extends FormComponentProps {
+  onSubmit: () => void;
+}
+
+type Ref = FormComponentProps;
+const FCForm = forwardRef<Ref, FCFormProps>(({ form, onSubmit }, ref) => {
```

Comment:
Code block should be simplify. Only show how to use `forwardRef` instead.

Reply:
I'll make an codesandbox instead of documenting it.